libraries/Microsoft.Bot.Builder.AI.QnA/AnswerSpanRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    public class AnswerSpanRequest
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets the enablet.
```

Comment:
>enablet [](start = 69, length = 7)

Is this a typo?

Reply:
Yes, fixed it now.

Reply:
fixed now.

---
In reply to: [433508289](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433508289) [](ancestors = 433508289)

Reply:
Thanks and fixed.

---
In reply to: [433508289](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433508289) [](ancestors = 433508289)